src/App.vue

```diff
@@ -115,8 +115,9 @@
               <header>
                 <h1>{{ getFilingName }}</h1>
               </header>
-              <p class="mt-4" v-if="getFilingSubtitle">
```

Comment:
This was overly complicated as it used an enum with a single key (FilingTypesSubTitle) and a getter with a single case.

We can make this complicated again in the future if we need to :)